src/resource_server/utils/signing_api.py

```diff
@@ -0,0 +1,62 @@
+from typing import Dict
+import requests
+import json
+
+from resource_server.utils.constants import BASE_URL 
+
+
+class SigningKey:
```

Comment:
what's the use of this class ?
not a single method uses `self` in a relevant way.